modular_skyrat/code/modules/surgery/surgery_step.dm

```diff
@@ -8,13 +8,17 @@
 		return FALSE
 	if(tool)
 		speed_mod = tool.toolspeed
+	
+	if(ishuman(target) && target.stat == CONSCIOUS && target.mob_biotypes & MOB_ORGANIC && !target.IsUnconscious() && !target.InCritical() && !target.IsSleeping() && !HAS_TRAIT(target, TRAIT_PAINKILLER))
+		to_chat(user, "<span class='warning'> They are currently concious and have had no painkillers. Are you sure you wish to continue?</span>")
```

Comment:
But it'll continue either way - not sure I like a question being asked like this in a chat warning.

Reply:
also *conscious

Reply:
Fair enough, I might make it a prompt (but the issue is the fact that it would cause things like tend wounds to spam people)

Reply:
I'd just have a punish mechanic so they mess it up in some way/person screams and doesn't auto continue.

Reply:
@Akrilla handled by existing code (It does brute and makes them scream)

Reply:
All this does is warn them /before/ they get to that stage in the surgery

Reply:
Then I'd remove this warning.

Reply:
fair